tests/intensive/sam_parity_tests.py

```
@@ -150,6 +174,25 @@ def _assert_parity(
     )
 
 
+def _assert_video_parity(test_case, dataset, pred_field, gt_field, eval_key):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_assert_video_parity`

Add return type annotation: `None`

(ANN202)